<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mslClass</class>
 <widget class="QWidget" name="mslClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MicroScope LightControl</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox_RGBLEDs">
     <property name="title">
      <string>RGB LEDs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="LayoutForColorSelector"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="verticalSlider_rgb">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Brightness</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_RGB_off">
        <property name="text">
         <string>RGB off</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QPushButton" name="pushButton_off">
     <property name="text">
      <string>Off</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" rowspan="2">
    <widget class="QGroupBox" name="groupBox_EEProm">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>EEPROM</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_EE_currentSetting">
        <property name="text">
         <string>To current setting</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeView" name="treeView"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_EE_Save">
        <property name="text">
         <string>SaveToEEProm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_EE_Read">
        <property name="text">
         <string>Read EEProm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4" rowspan="2">
    <widget class="QPushButton" name="pushButton_EEPROM">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Â»</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QGroupBox" name="groupBox_UVLEDs">
     <property name="title">
      <string>UV/WHITE LEDs</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBox_uv">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSlider" name="verticalSlider_uv">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>UV</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSlider" name="verticalSlider_white">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSpinBox" name="spinBox_white">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>WHITE</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QPushButton" name="pushButton_UvWhiteOff">
        <property name="text">
         <string>off</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QPushButton" name="pushButton_UV_White_Tggl">
        <property name="text">
         <string>&lt;&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QPushButton" name="pushButton_Wmax">
        <property name="text">
         <string>max</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QPushButton" name="pushButton_UVmax">
        <property name="text">
         <string>max</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_uv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_uv</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_uv</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_uv</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>251</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>verticalSlider_white</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_white</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_white</sender>
   <signal>valueChanged(int)</signal>
   <receiver>verticalSlider_white</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>282</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
